scripts/managedb/csubs_control/README.md

```diff
@@ -3,9 +3,6 @@ the same CLI arguments:
 
 * `-db <database>` (mandatory): to specify the DB to process
 * `--dry-run` (optional): it avoids actual modification in the DB, just reporting
-* `--with-restart <status_file>` (optional): restart Context Broker service if the database was manipulated and,
```

Comment:
Probably the scripts in this directory are no longer used, but I have tried to adapt them anyway.

NTC (informative)

Reply:
Tooling scripts are not covered by CNR. CNR refers only to changes in Orion Context Broker itself and some sidde artifacts like Dockerfile.

With regards to the removal of RPM itself, I think is was already covered in an implicit way in https://github.com/telefonicaid/fiware-orion/releases/tag/3.7.0

> Reference distribution changed RHEL/CentOS 8 to Debian 11

Reply:
Ok, NTC